### ISSUE TITLE
每次刷新都会跳到聊天的第一条信息

### ISSUE THREAD

**Issue by Jxells** [Kerwin1202/chatgpt-web#146]
如题。我记得以前的版本没有这个问题

**Comment by zhaobudaomingzi** [comment]
me too

**Comment by Jxells** [comment]
回退之前的版本没问题

**Comment by huangwb8** [comment]
有的人喜欢刷新跳第一条，有的人则喜欢跳最新的一条。 众口难调。

**Comment by Kerwin1202** [comment]
> 如题。我记得以前的版本没有这个问题

回退倒什么版本没问题，我前两天也遇到了，但是自己用的不是最新的，我昨天更新到最新好像没出现过

**Comment by Jxells** [comment]
> > 如题。我记得以前的版本没有这个问题
> 
> 回退倒什么版本没问题，我前两天也遇到了，但是自己用的不是最新的，我昨天更新到最新好像没出现过

我更新了今天的版本，依然会跳到第一条消息

**Comment by Urmi10799** [comment]
Yes this bug exists. On local run this bug isn't happening. but on remote host this bug occurred.

Here's what I found: 
When initially access the site that is hosted on remote server, this isn't occurred but after refreshing the page or close the browser and again go to site, this bug keeps appearing. 

Possible reason:
scrollToBotton implemented on **sider**, which is the main reason of this bug. Maybe there is a  chat data sync delay and during that time scrollToBottom is triggered.

**Comment by Urmi10799** [comment]
> > As the title. I remember that the previous version did not have this problem
> 
> It’s okay to go back to any version. I also encountered it two days ago, but I’m not using the latest version. I updated to the latest version yesterday and it didn’t seem to appear.

Yes this bug exists. On local run this bug isn't happening. but on remote host this bug occurred.

Here's what I found: 
When initially access the site that is hosted on remote server, this isn't occurred but after refreshing the page or close the browser and again go to site, this bug keeps appearing. 

Possible reason:
scrollToBotton implemented on **sider**, which is the main reason of this bug. Maybe there is a  chat data sync delay and during that time scrollToBottom is triggered.

**Comment by Jxells** [comment]
貌似这个统计功能也并未生效
![image](https://github.com/Kerwin1202/chatgpt-web/assets/94813798/312e3704-2510-4a9b-b82f-6cb9b2c2f7d8)

**Comment by Kerwin1202** [comment]
> 貌似这个统计功能也并未生效 ![image](https://user-images.githubusercontent.com/94813798/239774811-312e3704-2510-4a9b-b82f-6cb9b2c2f7d8.png)

新发一些聊天对话试试看 应该会有 旧的可能没有

**Comment by Jxells** [comment]
我发现我反代之后就会跳到第一条消息，不反代就是正常的

**Comment by Kerwin1202** [comment]
a6f7bc0ec12e4e8967697e2bf376ec1796f5851f  

这个提交尝试修复了下，如果有人自己 pull了，没问题 记得这里说下

I tried to fix this submission, but if someone pulled it themselves, there's no problem. Remember to mention it here

**Comment by Jxells** [comment]
> [a6f7bc0](https://github.com/Kerwin1202/chatgpt-web/commit/a6f7bc0ec12e4e8967697e2bf376ec1796f5851f)
> 
> 这个提交尝试修复了下，如果有人自己 pull了，没问题 记得这里说下
> 
> I tried to fix this submission, but if someone pulled it themselves, there's no problem. Remember to mention it here

如果数据库部署在另外的服务器上，这个问题依旧存在。当数据库和项目在同一服务器上，该问题的确得到了修复。
